### DISCUSSION TITLE
Roadrunner License.

### DISCUSSION THREAD

**Discussion by gouwylieven** [carla-simulator/carla#6433] [Q&A]
Hi dear people. I am incredibly happy I discovered about Carla. I am new and very eager to start working with it. as Carla seems to be working with Roadrunner alot, I am wondering, how can I get a Roadrunner License? I applied for one through the matlab site but as I am not a company, they did not reply until now... 

thank you so much for your help.

**Comment by Liuzy0908** [comment]
Get it from your university.

**Comment by gouwylieven** [comment]
i am not connected to a university. please tell me there is another way.

Sent from Yahoo Mail on Android 
 
  On Tue, Jul 18, 2023 at 11:15, Zhaoyang ***@***.***> wrote:   


Get it from your university.
